constructor/nsis/_nsis.py

```diff
@@ -229,6 +230,18 @@ def rm_regkeys():
             pass
 
 
+def win_del(dirname):
+    try:
+        out = check_output('DEL /F/Q/S {} > NUL'.format(dirname), shell=True, stderr=STDOUT, cwd=dirname)
```

Comment:
DEL operates on files, not on folders.  I think you need to make this just *.*, and rely on the cwd argument to put you in the right place.